### DISCUSSION TITLE
Weather widget not loading (Google cookie notice)

### DISCUSSION THREAD

**Discussion by Mark-Eng** [valinet/ExplorerPatcher#1301] [General]
Since today the weather widget is not working—only the generic "newspaper" symbol appears in the taskbar. This seems to have to do with Google's cookie/privacy notifications (see attached screenshot).

Any workarounds?

![image](https://user-images.githubusercontent.com/14117285/231405186-ae448af1-5b88-48c5-9089-604bc02fb2a7.png)

**Comment by JakeWerecat** [comment]
Huh, same trouble for me after installing KB5025239. It happened to me before every now and then, but fixed itself, this time is permanent though, and I can't get it to work :(

**Comment by markusb68** [comment]
same here:
![image](https://user-images.githubusercontent.com/20417704/231496570-0104c02b-8e3d-4392-b402-06087cb28cdc.png)

If I press reload it shows the weather and after one second it shows the above error page again.
BUT I don't see the Google's cookie/privacy notifications mentioned in post #1.

**Reply by jobeid** [reply]
Same issue here too. Just started today. Is everyone else seeing the weather? I'm wondering if the source is down.

**Comment by JakeWerecat** [comment]
For me the notification overlays the weather like this:

![Untitled](https://user-images.githubusercontent.com/130557724/231498703-221c6074-bf71-4c8c-82ed-9d2db928fc3a.png)
and then it goes to same screen as markusb68 showed.

**Comment by markusb68** [comment]
I switched on the dev mode in registry, made the weather window larger and pressed reload.
Then i can this for about one second. BUT I don't see the Google's cookie/privacy notifications mentioned in post #1
![image](https://user-images.githubusercontent.com/20417704/231681975-ab2b3f07-ceee-4be2-9b6a-b7f1e5ca50b2.png)

**Comment by sergyum** [comment]
Same trouble, after installing KB5025239, I think, or at least since then.

**Reply by God-damnit-all** [reply]
> Same trouble, after installing KB5025239, I think, or at least since then.

Has nothing to do with the KB, just coincidental timing.

**Comment by 0x00079** [comment]
Restarted my system to install the KB5025239 update and the widget got borked.
Hovering the widget icon keeps saying "No internet access", as reported pressing "Reload" will briefly show the widget in its normal state, then a Google search for "weather" and then the error "Unable to load weather information".

**Reply by God-damnit-all** [reply]
Has nothing to do with the KB, just coincidental timing.

**Comment by rm781** [comment]
Weather stopped loading with latest windows up date. OS build 22621.1555

**Comment by jw-mn** [comment]
https://github.com/valinet/ExplorerPatcher/issues/1310 
Temporary Fix
